hoover/search/ratelimit.py

```diff
@@ -29,6 +29,20 @@ def wrapper(request, *args, **kwargs):
             return view(request, *args, **kwargs)
         return wrapper
 
+    def get_user_limit(user):
```

Comment:
I thought this returns a single integer when looking at the two lines where it's used:
    
    return JsonResponse({
        'batch': settings.HOOVER_BATCH_LIMIT,
        'requests': get_user_limit(request.user),
    })

Name it so I get it that this returns a dict with lots of data, like `get_user_limit_data` or maybe just `get_user_limits`

Reply:
Yeah, that was a lousy name, I think initially it did return an integer. There, I've changed it.